src/components/WalletModal/Option.tsx

```diff
@@ -33,11 +49,18 @@ const OptionCardLeft = styled.div`
   height: 100%;
 `
 
-const OptionCardClickable = styled(OptionCard as any)<{ clickable?: boolean }>`
+const OptionCardClickable = styled(OptionCard as any)<{
+  active?: boolean
+  clickable?: boolean
+  redesignFlag?: boolean
+}>`
   margin-top: 0;
+  border: ${({ active, theme }) => (active ? `1px solid ${theme.accentActive}` : ``)};
```

Comment:
i think this can be active && `1px solid ${theme.accentActive}` so you don't have a blank

Reply:
sgtm done